prime-router/src/main/kotlin/fhirengine/engine/Message.kt

```diff
@@ -58,6 +58,7 @@ data class RawSubmission(
     val blobURL: String,
     val digest: String,
     val blobSubFolderName: String,
+    val topic: String,
```

Comment:
Considering making this change at least a temporary way to account for the transition period where messages in the queue might not have a topic.
```suggestion
    val topic: String = "",
```